### ISSUE TITLE
Implement schema list page

### ISSUE THREAD

**Issue by kgodey** [centerofci/mathesar#171]
**Problem**

Users should be able to create a new schema, edit schema names, and delete schemas.

**Proposed solution**

We should provide a way to do these actions from the UI using the schema list page introduced in the [design spec](https://wiki.mathesar.org/en/design/specs/schemas).

**Additional context**

- #166
- #168 
- #170
- #393

**Comment by kgodey** [comment]
This will be unblocked once #393 is merged.

**Comment by pavish** [comment]
As part of this issue, a new route has to be created for schema listing, and only the content of schema listing is to be developed. We have another issue https://github.com/centerofci/mathesar/issues/362 which handles the changes in the top navigation bar.

In order to facilitate development independently, we can only have a placeholder button to navigate to the schema listing page, as part of the PR for this issue.

**Comment by kgodey** [comment]
@dhruvkb How's this coming? Do you need any help?

**Comment by dhruvkb** [comment]
I'm working on it. I should be able to wrap this up during the upcoming weekend as I didn't get much time during the last.

**Comment by kgodey** [comment]
Sounds good, thanks!
